zio-http/src/test/scala/zio/http/TestServer.scala

```diff
@@ -7,22 +7,17 @@ import zio.http.Server.ErrorCallback
  * Enables tests that make calls against "localhost" with user-specified
  * Behavior/Responses.
  *
- * @param state
- *   State that can be consulted by our behavior PartialFunction
  * @param behavior
  *   Describes how the Server should behave during your test
  * @param driver
  *   The web driver that accepts our Server behavior
  * @param bindPort
  *   Port for HTTP interactions
- * @tparam State
- *   The type of state that will be mutated
  */
-final case class TestServer[State](
-  state: Ref[State],
-  behavior: PartialFunction[(State, Request), (State, Response)],
+final case class TestServer(
+  behavior: Ref[PartialFunction[Request, ZIO[Any, Nothing, Response]]],
   driver: Driver,
-  bindPort: Int,
+  bindPort: RuntimeFlags,
```

Comment:
Bah, I knew that Intellij would incorrectly suggest `RuntimeFlags` in place of `Int` when generating type annotations, but this is a new manifestation of that issue-

If you use the "Change Signature" refactoring and modify a parameter, it will automatically "fix" any existing `Int` parameter by changing it to `RuntimeFlags`.

https://user-images.githubusercontent.com/2054940/194116555-188129d9-4078-4f06-a078-a92a04971ca5.mov

Looks like I triggered that when I was removing `State` from `TestServer`